### ISSUE TITLE
Improper Wav Format For Master File

### ISSUE THREAD

**Issue by carrickr** [nulib/avalon#363]
Media Object: https://media.northwestern.edu/media_objects/j9602133w

Part 17 is not in a format supported by Amazon.  For `.wav` the codec has to be `pcm`.  If the file is converted I can upload the new master file to S3 and just rerun that particular master file.  I'm also happy to convert it via quicktime on my machine if someone knows the desired output quality.

**Comment by davidschober** [comment]
@laura-alagna can you grab this? I don't have the original.

**Comment by laura-alagna** [comment]
@davidschober yup I can take care of it

**Comment by carrickr** [comment]
@laura-alagna 

At the end of this we had 41 total:

```
"bienen0278_01.wav", "bienen1003_17.wav", "bienen1985_07.wav", "bienen0929_05.wav", "bienen1218_02.wav", "bienen0622_02.wav", "bienen0932_12.wav", "bienen1004_01.wav", "bienen1985_06.wav", "bienen1226_02.wav", "bienen1433_06.wav", "bienen1003_15.wav", "bienen1004_04.wav", "bienen1433_08.wav", "bienen0932_09.wav", "bienen0391_02.wav", "bienen1003_13.wav", "bienen0936_06.wav", "bienen0117_13.wav", "bienen1003_14.wav", "bienen1003_11.wav", "bienen1003_08.wav", "bienen1199_03.wav", "bienen1826_03.wav", "bienen1826_02.wav", "bienen1004_05.wav", "bienen1004_03.wav", "bienen0936_07.wav", "bienen1985_08.wav", "bienen1985_03.wav", "bienen1985_05.wav", "bienen1003_12.wav", "bienen2204_02.wav", "bienen1985_09.wav", "bienen0932_08.wav", "bienen1119_05.wav", "bienen1003_16.wav", "bienen1003_10.wav", "bienen0936_03.wav", "bienen1004_08.wav"```

**Comment by laura-alagna** [comment]
@carrickr 

we can ignore the following:
bienen0329_09.wav
bienen1229_05.wav
bienen1538_10.wav
bienen1812_05.wav
bienen2135_03.wav
bienen2450_06.wav

0329 = no track 9 exists
1229 = no track 5 exists
1538 = no track 10 exists
1812 = no track 5 exists
2135 = 4 second track; ignore (recording error or equipment test)
2450 = 4 second track

**Comment by laura-alagna** [comment]
@carrickr I investigated the codec errors. Sadly some of them are corrupted files on CD (this collection is 15-25 year old CDs, so to be expected to some extent). 

But I was able to get 9 of the 40 bad codec files! I added those on dl17, just needs a resync to get them in I believe. 

they are:
bienen0622_02.wav
bienen0932_08.wav
bienen0932_09.wav
bienen1119_05.wav
bienen1199_03.wav
bienen1218_02.wav
bienen1826_02.wav
bienen1826_03.wav
bienen2204_02.wav
